content/servicemesh/cleanup.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Cleanup"
+date: 2018-11-13T23:59:44+09:00
+weight: 70
+draft: false
+---
+
+#### Remove telemetry configuration / port-forward process
+
+```
+kubectl delete -f istio-telemetry.yaml
+
+killall kubectl
+```
+
+#### Remove the application virtual services / destination rules
+
+```
+kubectl delete -f samples/bookinfo/networking/virtual-service-all-v1.yaml
+
+kubectl delete -f samples/bookinfo/networking/destination-rule-all.yaml
+```
+
+#### Remove the gateway / application
+
+```
+kubectl delete -f samples/bookinfo/networking/bookinfo-gateway.yaml
+
+kubectl delete -f <(istioctl kube-inject -f samples/bookinfo/platform/kube/bookinfo.yaml)
```

Comment:
istioctl is not necessary to delete the sample app pods. so, I will remove the command. and istioctl supposed to be moved into /usr/local/bin if you ran the workshop from the first, not sure if you set the PATH for istio-1.0.3/bin before and ran the command in other directory then it might fail.